### ISSUE TITLE
Failing to load liveboard and worksheet tmls after v10.5.0.cl was released

### ISSUE THREAD

**Issue by benjiro** [thoughtspot/thoughtspot_tml#25]
After our cloud instance was upgraded to 10.5.0.cl the following two errors occur when trying load tml from `loads(tml_document)`

Liveboards
`PinboardEDocProto.__init__() got an unexpected keyword argument 'ordered_chips'`

Worksheets
`WorksheetEDocProtoQueryProperties.__init__() got an unexpected keyword argument 'sage_config'`

Tables
`LogicalTableEDocProto.__init__() got an unexpected keyword argument 'properties'`

Workaround is to drop the sections from the string before trying to load them in.

Python 3.13.1
thoughtspot_rest_api_v1 1.8.2
thoughtspot_tml 2.1.1

**Comment by boonhapus** [comment]
I'll build a new version from source and release it within a few days @benjiro , thanks for this report!

**Comment by boonhapus** [comment]
@benjiro upgrading to `thoughtspot_tml == 2.2.0` should help.. though I wasn't able to find `sage_config` **[in the new spec](https://github.com/thoughtspot/thoughtspot_tml/commit/8e881ec47a6eb564ef3267bbcd9e146e2054f057)**.

If you're comfortable attach the example TML here, or shooting it over to me via email, I can investigate a bit further.

**Comment by benjiro** [comment]
@boonhapus no problems it's fixed two of the problems that sage config is still a issue. I've emailed you the TML of the worksheet that has the new properties in it as an example.

**Comment by boonhapus** [comment]
Thanks @benjiro -- I was able to isolate the issue and fix it. Can you try now with **[v2.2.1](https://github.com/thoughtspot/thoughtspot_tml/releases/tag/v2.2.1)** ?

**Comment by benjiro** [comment]
@boonhapus Yup that has resolved the last issue, thanks for getting it fixed 👍🏻
